.github/workflows/clangtidy_review.yml

```diff
@@ -0,0 +1,35 @@
+name: Clang tidy Review
+
+on: [pull_request]
+
+jobs:
+  check:
+    name: Clang tidy Review 
+    runs-on: ubuntu-18.04
```

Comment:
Let's run it on ubuntu 20.04

Reply:
From Gitter chat:

> I have upgraded the machines to use ubuntu 20.04 on all Linux instances except for clang-tidy check Job, it still uses ubuntu 18.04 but uses clang-tidy-10. I could not upgrade it because on ubuntu 20.04 the multi-precision library does not compile with clang-10. Maybe in future when new boost releases are made, we will upgrade it to 20.04. See this, I tried to upgrade but got [this error](https://github.com/BoostGSoC20/ublas/runs/775558213?check_suite_focus=true#step:5:65)

We do use multi-precision in one of the Unit Tests! It is that error.

Reply:
Yes. You are right. The reason why I included multiprecision is that I wanted to make sure that we can have also more complex types supporting our library. It is not that crucial. we could also think of including `std::ratio` or `std::complex` instead.
Hence my suggestion: comment out multiprecision and include `std::complex` and **use ubuntu 20.04**

Reply:
Okay, Got it!

Reply:
No, need. Seems like this issue has been fixed in `boost/multiprecision` master